spec/server/unit/locale.spec.coffee

```diff
@@ -0,0 +1,91 @@
+english = require("../../../app/locale/en")
+langs = [
```

Comment:
Possible to do this without enumeration? It's already pretty error-prone to add new languages in the 3 places we have to make edits.